<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetPixelScope</class>
 <widget class="QWidget" name="WidgetPixelScope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PixelScope</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBoxAutoScale">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Turn on/off auto scaling. If auto scale is on the axis range is floating depending from current minimal and maximal signal level.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Auto range</string>
       </property>
      </widget>
     </item>
     <item row="1" column="11">
      <widget class="QCheckBox" name="checkBoxShowDcs2">
       <property name="toolTip">
        <string>show DCS2 curve</string>
       </property>
       <property name="text">
        <string>DCS2</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="10">
      <widget class="QSpinBox" name="spinBoxRightMin">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Minimum value for amplitude axis&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>200000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="12">
      <widget class="QCheckBox" name="checkBoxShowDcs3">
       <property name="toolTip">
        <string>show DCS3 curve</string>
       </property>
       <property name="text">
        <string>DCS3</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="10">
      <widget class="QCheckBox" name="checkBoxShowDcs1">
       <property name="toolTip">
        <string>show DCS1 curve</string>
       </property>
       <property name="text">
        <string>DCS1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLabel" name="labelLeftMax">
       <property name="text">
        <string>max  </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QSpinBox" name="spinBoxLeftMax">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Maximum value for distance or amplitude axis&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>2000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="lMeasurementType">
       <property name="text">
        <string>Distance [mm]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="8">
      <widget class="QSpinBox" name="spinBoxRightMax">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Maximum value for amplitude axis&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>3000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>3000</number>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QLabel" name="labelRightMax">
       <property name="text">
        <string>max</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="checkBoxShowLeft">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>distance</string>
       </property>
      </widget>
     </item>
     <item row="2" column="13">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="labelLeftMin">
       <property name="text">
        <string>min  </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QCheckBox" name="checkBoxShowDcs0">
       <property name="toolTip">
        <string>show DCS0 curve</string>
       </property>
       <property name="text">
        <string>DCS0</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QCheckBox" name="checkBoxShowRight">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>amplitude</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="2">
      <widget class="QSpinBox" name="spinBoxLeftMin">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Minimum value for distance or amplitude axis&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="3" column="11">
      <widget class="QLabel" name="labelRightMin">
       <property name="text">
        <string>min</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="8">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="rbDCS0">
         <property name="text">
          <string>DCS0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbDCS1">
         <property name="text">
          <string>DCS1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbDCS2">
         <property name="text">
          <string>DCS2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbDCS3">
         <property name="text">
          <string>DCS3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="0"/>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="labelActValue">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="labelCoordinates">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select pixels: Ctrl + Alt + left mouse button&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Deselect: Ctrl + Alt + right mouse button&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Deselect all: right mouse button&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Select pixels: Ctrl + Alt + left mouse button</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
